.travis.yml

```diff
@@ -13,10 +13,11 @@ after_success:
 matrix:
   fast_finish: true
   include:
-  - python: '3.8'
+  - name: lint
+    python: '3.8'
     install: pip install flake8
     script: make lint
-  include:
-  - python: '3.8'
+  - name: format
+    python: '3.8'
     install: pip install black==20.8b1
```

Comment:
This can be removed now.

Reply:
Sorry, not following what you want to remove. Do you want Travis to run:

A. Two separate code-checking jobs: "lint" (flake8) and "format" (black)?
B. Only one code-checking job that includes both flake8 and black?
C. Neither?

A is what's in this PR ([build output](https://travis-ci.org/github/syrusakbary/snapshottest/builds/731710336))

Reply:
We don't need to run `pip install black==20.8b1` here because you've added it to `make install`.

It had been installed here, instead of through `make install`, because black==2.0.8b1 can't be installed under Python 2.7.

Reply:
Got it. But, um, see [my downstream comment](https://github.com/syrusakbary/snapshottest/pull/135#issuecomment-701618483). We still need to split apart linting/formatting install from testing install, one way or another.